tests/test.xpi.js

```diff
@@ -0,0 +1,238 @@
+import Xpi from 'xpi';
+import { Readable } from 'stream';
+
+
+const defaultData = {
+  compressionMethod: 8,
```

Comment:
This is based on a sample of the data we get back from yauzl. So we're not defining that. But 8 means deflate, 0 means no compression. I left this is because I think we'll probably want to explode if it's not 8 or 0 as they are the only supported values.